pyincore/analyses/mlenabledcgeslc/mlcgeslc.py

```diff
@@ -105,7 +105,8 @@ def run(self) -> bool:
         for sector in self.cap_shock_sectors:
             if sector.upper() not in [v.upper() for v in sector_shocks["sector"]]:
                 raise ValueError(
-                    f"Sector {sector} not found in the sector shocks file with\n {sector_shocks['sector']} sectors. \nPlease make sure you have used the correct capital shocks"
+                    f"Sector {sector} not found in the sector shocks file with\n {sector_shocks['sector']} sectors. 
+                    \nPlease make sure you have used the correct capital shocks"
```

Comment:
There's an error here now. I think the line \n needs to start with f"

Reply:
I think now it should be good.